tests/toranj/start.sh

```diff
@@ -81,23 +81,7 @@ cd ../..
 # Build OpenThread posix mode with required configuration
 
 ./bootstrap || die
-./configure                             \
-    CPPFLAGS='-DOPENTHREAD_PROJECT_CORE_CONFIG_FILE=\"../tests/toranj/openthread-core-toranj-config.h\"' \
-    --enable-ncp-app=all                \
-    --with-ncp-bus=uart                 \
-    --with-examples=posix               \
-    --enable-border-router              \
-    --enable-child-supervision          \
-    --enable-diag                       \
-    --enable-jam-detection              \
-    --enable-legacy                     \
-    --enable-mac-filter                 \
-    --enable-service                    \
-    --enable-channel-monitor            \
-    --disable-docs                      \
-    --disable-test || die
-
-make -j 8 || die
+COVERAGE=1 make -f examples/Makefile-posix all || die
```

Comment:
Does a `COVERAGE=1 make -j 8` works here?

Reply:
`BuildJobs` defaults to 10 in [examples/Makefile-posix](https://github.com/openthread/openthread/blob/master/examples/Makefile-posix#L42).

Reply:
(I don't mean about the build jobs)
I meant if the `COVERAGE=1` will enable code coverage...will "COVERAGE=1 make" work and enable coverage?... :)

Reply:
I tried that but ran into the following issue:
```
File 'main.c'
Lines executed:88.89% of 18
Branches executed:100.00% of 4
Taken at least once:50.00% of 4
Calls executed:80.00% of 10
Creating 'main.c.gcov'
Cannot open source file main.c
File 'misc.c'
Lines executed:10.53% of 19
Branches executed:0.00% of 2
Taken at least once:0.00% of 2
Calls executed:0.00% of 5
Creating 'misc.c.gcov'
Cannot open source file misc.c
File 'flash.c'
Lines executed:85.11% of 47
Branches executed:88.24% of 34
Taken at least once:47.06% of 34
Calls executed:81.82% of 11
Creating 'flash.c.gcov'
Cannot open source file flash.c
```

Compare that to using `Makefile-posix`
```
File '/home/travis/build/jwhui/openthread/examples/apps/ncp/main.c'
Lines executed:84.62% of 13
Branches executed:100.00% of 4
Taken at least once:50.00% of 4
Calls executed:77.78% of 9
Creating '#home#travis#build#jwhui#openthread#examples#apps#ncp#main.c.gcov'
File '/home/travis/build/jwhui/openthread/examples/apps/cli/main.c'
No executable lines
No branches
No calls
Removing '#home#travis#build#jwhui#openthread#examples#apps#cli#main.c.gcov'
File '/home/travis/build/jwhui/openthread/examples/platforms/posix/logging.c'
Lines executed:0.00% of 9
Branches executed:0.00% of 6
Taken at least once:0.00% of 6
Calls executed:0.00% of 1
Creating '#home#travis#build#jwhui#openthread#examples#platforms#posix#logging.c.gcov'
```

Maybe it doesn't matter.  I'll see if codecov.io can deal with it.

Reply:
Pushed an update.  Let's see how it does.

Reply:
OK, looks like codecov.io can still handle it ([results](https://codecov.io/gh/openthread/openthread/pull/2654/tree)).